citylearn/data/citylearn_2022_phase_all_plus_evs/schema.json

```diff
@@ -0,0 +1,1034 @@
+        }
+      }
+    },
+    "EV_3": {
+      "include": true,
+      "energy_simulation": "EV_3.csv",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "image_path": "images/ev.png",
+      "battery": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 75,
+          "nominal_power": 50,
+          "initial_soc": 0.5
+        }
+      }
+    },
+    "EV_4": {
+      "include": true,
+      "energy_simulation": "EV_4.csv",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "image_path": "images/ev.png",
+      "battery": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 45,
+          "nominal_power": 50,
+          "initial_soc": 0.2
+        }
+      }
+    },
+    "EV_5": {
+      "include": true,
+      "energy_simulation": "EV_5.csv",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "image_path": "images/ev.png",
+      "battery": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 90,
+          "nominal_power": 50,
+          "initial_soc": 0.8
+        }
+      }
+    },
+    "EV_6": {
+      "include": true,
+      "energy_simulation": "EV_6.csv",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "image_path": "images/ev.png",
+      "battery": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 100,
+          "nominal_power": 50,
+          "initial_soc": 0.8
+        }
+      }
+    },
+    "EV_7": {
+      "include": true,
+      "energy_simulation": "EV_7.csv",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "image_path": "images/ev.png",
+      "battery": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 60,
+          "nominal_power": 50,
+          "initial_soc": 0.1
+        }
+      }
+    },
+    "EV_8": {
+      "include": true,
+      "energy_simulation": "EV_8.csv",
+      "inactive_observations": [],
+      "inactive_actions": [],
+      "image_path": "images/ev.png",
+      "battery": {
+        "type": "citylearn.energy_model.Battery",
+        "autosize": false,
+        "attributes": {
+          "capacity": 80,
+          "nominal_power": 50,
+          "initial_soc": 0.2
+        }
+      }
+    }
+  },
+  "buildings": {
+    "Building_1": {
+      "include": true,
+      "energy_simulation": "Building_1.csv",
+      "weather": "weather.csv",
+      "carbon_intensity": "carbon_intensity.csv",
+      "pricing": "pricing.csv",
+      "coordinates": {
+        "latitude": 41.195632,
+        "longitude": -8.598365
+      },
+      "image_path": "images/building.png",
+      "reward_type": "D",
```

Comment:
Reward Type was one variable I introduced to identify the type of prosumer objective for that house in particular. In other words, in my project each house could chose from one different objective. It can be taken out now

Reply:
I took that out in the merge code